third-party/karpenter/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Change Log
```

Comment:
Changelog will be generated automatically. You don't have to create one by yourself.